docs/installation.md

```diff
@@ -1,8 +1,10 @@
 # Installation 
 
+This guide helps you install KLoadGen plugin.
+
 ## Install as a JMeter library
 
-By default, the installation process will generate the JMeter plugin jar file. After cloning the repository, run the following command:
+By default, the installation process will generate the JMeter plugin JAR file. After cloning the repository, run the following command:
```

Comment:
Where the user has to run the command? In the repository root folder? I think it is a good idea to specify it. The user has recently arrived at the product, and he/she needs specific instructions.

Reply:
Both this command and the above dependency are part of Maven. People using this plugin should be familiar with Maven, I guess that's why this is not specified.
This command must be executed in the root of the project, where the pom.xml is located.

Reply:
Done!